pkg/validation/telemetry_validation.go

```diff
@@ -1,4 +1,4 @@
-package telemetry
+package validation
```

Comment:
Why the package change?

Reply:
I'll let @Miles-Garnsey describe the intent here.  I can see where having our validations all in one `validation` package could be handy for generalizing and refactoring in the future.

Reply:
The package change is required to avoid an import cycle due to the webhooks being located in `apis`.